package.json

```diff
@@ -272,6 +272,7 @@
 	"scripts": {
 		"build": "npm run build:packages && wp-scripts build",
 		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
+		"build:blocks-manifest": "wp-scripts build-blocks-manifest",
```

Comment:
This could go next in another PR to make it all simpler to land 😄

Reply:
This could be removed for now as it isn’t connected anywhere.

Reply:
Thank you, removed here: e24747ee85